tests/ui/ReportDetailsPageTest.tsx

```diff
@@ -85,6 +86,7 @@ describe('ReportDetailsPage', () => {
                 </LocaleContextProvider>
             </OnyxListItemProvider>,
         );
+        await waitForBatchedUpdates();
```

Comment:
Why after this PR we need to wait for batched updates here?

Reply:
To wait for translations (they're happending in the LocaleContextProvider)